### DISCUSSION TITLE
Re: Make 'Segments to chapters' only export the SELECTED segments' #2157:

### DISCUSSION THREAD

**Discussion by cagrey** [mifi/lossless-cut#2181] [Support]
I am having the same problem: Lossless-cut USED to work the desired way (export ONLY the selected segments) for ever - at least to my previous downloaded version (~maybe 3.56.n), without that orange "!" warning. Now I am sorry that I've deleted that old version after updating to 3.62, without checking the results first :(

Could you please go back to the original behavior (export only selected segments)? It doesn't make much sense to me either to
- export all the segments, selected or not (what I did NOT select is NOT a segment to me); or
- squash all the selected segments/chapters into one after export.
  
I used to select segments out of my various videos (excluding / deleting the useless shots), and then use them in new combinations - always worked well until recently. The new "improvement" makes my workflow much more difficult (export segments separately, find what I need in a huge list of individual segments...) :(

But thanks for the great tool anyways ;-)

**Comment by mifi** [comment]
#2157
